src/hist.jl

```diff
@@ -96,7 +105,7 @@ function histrange(lo::F, hi::F, n::Integer, closed::Symbol=:left) where F
             len += one(F)
         end
     end
-    Base.floatrange(start,step,Int(len),divisor)
+    _floatrange(start,step,Int(len),divisor)
```

Comment:
Since this is only used in one place, we could opt to inline the definition:
```suggestion
    StepRangeLen(Base.TwicePrecision{Float64}((start, divisor)),
                 Base.TwicePrecision{Float64}((stop, divisor)),
                 Int(len))
```